CHANGELOG.md

```diff
@@ -4,6 +4,24 @@ This file contains notable changes (e.g. breaking changes, major changes, etc.)
 
 This file was introduced starting Kani 0.23.0, so it only contains changes from version 0.23.0 onwards.
 
+## [0.29.0]
+
+### What's Changed
+* Fix symtab json file removal and reduce regression scope ([pull request](https://github.com/model-checking/kani/pull/2447) by @celinval)
+* Fix regression on concrete playback inplace ([pull request](https://github.com/model-checking/kani/pull/2454) by @celinval)
+* Update rust toolchain to 2023-04-29 ([pull request](https://github.com/model-checking/kani/pull/2452) by @zhassan-aws)
+* Update stubbing.md with the right command ([pull request](https://github.com/model-checking/kani/pull/2460) by @jaisnan)
+* Create a playback command to make it easier to run Kani generated tests ([pull request](https://github.com/model-checking/kani/pull/2464) by @celinval)
```

Comment:
I wonder if we should add this as a major change. I don't think we have a guideline regarding new subcommands.